src/scantailor/app/ui/dialogs/about.py

```
@@ -2,20 +2,24 @@
 from scantailor import __about__
-from scantailor.app.ui import UI_FOLDER, load_ui_widget
+from scantailor.app.ui import load_ui_widget
 from scantailor.app.ui.utils import get_cwidget
 
+_UI_FOLDER = Path(__file__).parent
+
 
 class AboutDialog(QtWidgets.QDialog):
     """About Dialog."""
 
     def __init__(self, parent: QtWidgets.QWidget | None = None):
```

Ruff (0.15.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)